procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+    2. Once these changes are CR'd and merged, publish a new minor version of each of these packages
+        1. Make PRs: [example `pushapkscript` release PR](https://github.com/mozilla-releng/pushapkscript/pull/70)
+        2. Once these release PRs are CR'd and merged, publish each package:
+            1. `git checkout master`, make sure you're on the "Merge pull request" commit for your PR
+            2. `git tag $version`, e.g.: `git tag 0.11.0`
+            3. `git push --tags && git push upstream --tags` (assuming that the `origin` remote is for your fork, and `upstream` is the `mozilla-releng` repo)
+            4. Ensure you're in the Python virtual env for your package (One approach is to share a single virtual env between all scriptworker repos)
+            5. `rm -rf dist && python setup.py sdist bdist_wheel` build the package
+            6. Publish to `Pypi`:
+                1. `gpg --list-secret-keys --keyid-format long` to get your GPG identity (it's the bit after "sec   rsaxxxx/"). An example GPG identity would be `5F2B4756ED873D23`
+                2. `twine upload --sign --identity $identity dist/*` to upload to Pypi (you may need to `pip install twine` first)
+
+10. Update configuration in [`build-puppet`](https://github.com/mozilla-releng/build-puppet/)
+    1. Locate signing secrets (there's six different items you'll add to `hiera`)
+        1. You should've received signing credentials from step 1. Print out the decrypted file you received: `gpg -d <file from step 1>`
+        2. We will want to encrypt the "dep" and "rel" credentials for the "prod" autograph instance. They can be identified as lines that contain a "list" where the second item ends with "_dep" or "_prod", respectively
+            * Example: "dep" line would be: `["http://<snip>", "signingscript_fenix_dep", "<snip>", ["autograph_apk"], "autograph"]`
+        3. For these two lines, the secrets we want to put in hiera are the username and password (the second and third item)
+        4. Later, in step 19, you'll have been emailed a Google Play service account and key. However, for now, we're going to use a dummy value (the string "dummy") as placeholders for these values
+
+    2. Add secrets to `hiera`
+        1. Connect to VPN
+        2. SSH into `releng-puppet2.srv.releng.mdc1.mozilla.com`
+        3. `sudo cp /etc/hiera/secrets.eyaml /etc/hiera/secrets.eyaml.$username` (substituting your username in) to back up the hiera secrets file
+        4. For each secret, encrypt it with `sudo eyaml encrypt --pkcs7-private-key /etc/hiera/keys/private_key.pem --pkcs7-public-key /etc/hiera/keys/public_key.pem --output examples -p -l '$lookupkey'` (it prompts you to paste the secret). The "lookup key" is different for each secret we put in Hiera:
```

Comment:
Nit: Might be worth to also pointing to [this](https://wiki.mozilla.org/ReleaseEngineering/PuppetAgain/Secrets#Using_EYAML) for more docs on this matter?

Reply:
I'll link to the docs, but I'll still inline the commands so that it's easy to see what needs to be accomplished